1. Contributing/Google Summer of Code/README.md

```diff
@@ -16,11 +16,11 @@ For general information, and some real-time help on next steps, please visit our
 * **Goal/Deliverable:** The foundation for all Rocket.Chat users/bots/services to optionally communicate with one another, across all connected instances.
 
 
-### Improved Off The Record Messaging (OTR)
+### Improved End-to-End Encryption Implemenation (currently feature-named OTR)
 * **Mentors:** Marcelo Schmidt ([@marcelo.schmidt](https://github.com/marceloschmidt))  Mitar Milutinovic ([@mitar](https://github.com/mitar))
-* **Description:** Taking and improving our OTR implementation
-* **Desirable Skills:** Some javascript coding skills and familiarity with cryptography.
-* **Goal/Deliverable:** The ability to have secure OTR conversations
+* **Description:** Taking and improving our end-to-end encryption implementation. Perhaps replace with more modern protocols, such as Signal. 
+* **Desirable Skills:** Some Javascript coding skills and familiarity with cryptography.
+* **Goal/Deliverable:** The ability to have secure conversations
```

Comment:
So this is a question how open we want this for students. One option would be to simply say, we want signal protocol. If we do not specify it so precisely, then we should probably be more precise here and say something out of many possibilities like  that we want secure conversations between two parties, in group chats, when all parties are online, when some are offline, secure file transfers, and so on.